sdk/core/azure-core/inc/azure/core/base64.hpp

```diff
@@ -38,6 +39,21 @@ namespace Azure { namespace Core {
      */
     static std::string Base64Encode(const std::vector<uint8_t>& data);
 
+    /**
+     * @brief Encodes the vector of binary data into UTF-8 encoded text represented as Base64.
+     *
+     * @param data The binary data to be encoded.
+     * @param length The length of the binaryData parameter.
+     * @return The UTF-8 encoded text in Base64.
+     */
+    static std::string Base64Encode(uint8_t const* const data, size_t length);
+
+    template <size_t length>
+    static std::string Base64Encode(std::array<uint8_t, length> const& data)
```

Comment:
From offline discussion: This API is purely convenience. To reduce public surface area in core that isn't needed by a customer ask, consider hiding this for now as a private implementation detail within websockets.

Reply:
Actually removed both overrides - when I wrote the code, I needed both of them, after refactoring, it turns out they aren't needed any more.